internal/provider/resource_project_cloud_account_link.go

```diff
@@ -0,0 +1,668 @@
+package provider
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/google/uuid"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"wiz.io/hashicorp/terraform-provider-wiz/internal"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/client"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/utils"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/wiz"
+)
+
+// CloudAccountSearchResponse represents the response from a cloud account search.
+// It includes a GraphSearch object that contains a list of Nodes, each with a list of Entities.
+type CloudAccountSearchResponse struct {
```

Comment:
The vendor schema is defined in ./internal/wiz/structs.go and there is already a model for the response defined (which should be of type `GraphSearchResultConnection` [here](https://github.com/AxtonGrams/terraform-provider-wiz/blob/main/internal/wiz/structs.go#L1736)

Reply:
I tried to use that type but the `GraphSearch` field is missing and the response isn't deserialized properly. I will move the type to ./internal/wiz/structs.go.